concepticondata/conceptlists/Williamson-1973-118.tsv

```diff
@@ -0,0 +1,119 @@
+ID	NUMBER	ENGLISH	FRENCH	GERMAN	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Williamson-1973-118-1	1	animal	animal	Tier	619	ANIMAL
+Williamson-1973-118-2	2	arm: hand	bras: main	Arm: Hand	2121	ARM OR HAND
+Williamson-1973-118-3	3	armpit	aisselle	Achselhöhle	1886	ARMPIT
+Williamson-1973-118-4	4	arrow	flèche	Pfeil	977	ARROW
+Williamson-1973-118-5	5	ashes	cendres	Asche	646	ASH
+Williamson-1973-118-6	6	ask (a question)	demander	fragen	1309	ASK (INQUIRE)
+Williamson-1973-118-7	7	bad, be	mauvais (être)	schlecht sein	1292	BAD
+Williamson-1973-118-8	8	bean	haricot	Bohne	832	BEAN
+Williamson-1973-118-9	9	beg (=ask for something)	demander (qqc à qqn)	bitten um, erbitten, betteln	1828	ASK (REQUEST)
```

Comment:
We also have 3534	BEG. The glosses in German refer to both 1828 ASK (REQUEST) and 3534 BEG. Should we unmap it?

Reply:
I'd vote for leaving it as it is. If German was not the original language for this list but rather added as translation, I'd say we should focus on the English glosses. Apart from that, BEG in the Concepticon denotes the way of living rather than a single act of asking for something (which is not necessarily money or food).

Reply:
I second this. "Betteln" can also refer to people asking for something outside of the way-of-living begging so all glosses seem to align with the "asking for" meaning.